packages/client/hmi-client/src/components/workflow/util.ts

```diff
@@ -4,9 +4,43 @@ import type { CsvAsset, TimeSpan } from '@/types/Types';
 import type { WorkflowNode } from '@/types/workflow';
 import type { CalibrateMap } from '@/services/calibrate-workflow';
 import { useProjects } from '@/composables/project';
+import { getModelConfigurationById } from '@/services/model-configurations';
+import { getModel } from '@/services/model';
+import { Model } from '@/types/Types';
+
+// FIXME: replace with single API call
+export const getModelByModelConfigurationId = async (id: string) => {
+	const modelConfig = await getModelConfigurationById(id);
+	const model = await getModel(modelConfig.modelId as string);
+	return model;
+};
```

Comment:
You can use the new endpoint `/models/from-model-configuration/{model-config-id}` now.

Reply:
done thx